intake_elasticsearch/elasticsearch_table.py

```diff
@@ -57,16 +56,25 @@ def _get_schema(self):
         return base.Schema(datashape=None,
                            dtype=dtype,
                            shape=shape,
-                           npartitions=1,
+                           npartitions=self.npartitions,
                            extra_metadata=self.metadata)
 
     def to_dask(self):
         """Make single-partition lazy dask data-frame"""
         import dask.dataframe as dd
         from dask import delayed
         self.discover()
-        part = delayed(self._get_partition(0))
-        return dd.from_delayed([part], meta=self.dtype)
+        parts = []
+        if self.npartitions == 1:
+            part = delayed(self._get_partition(0))
+            parts.append(part)
```

Comment:
`parts = [delayed(self._get_partition(0))]`; in practice, `to_dask()` is not useful when there's only one partition, so this may never get used.

Reply:
Didn't know if you wanted previous functionality. Will remove it. Much clearer.